demos/serverless/src/handlers.js

```diff
@@ -13,6 +13,7 @@ const ddb = new AWS.DynamoDB();
 const endpoint = process.env.CHIME_ENDPOINT;
 const currentRegion = process.env.REGION;
 const useChimeSDKMeetings = process.env.USE_CHIME_SDK_MEETINGS;
```

Comment:
Didn’t understand the first part. This is a boolean variable and it will always have a default value.

Reply:
useChimeSDKMeetings  should be true if we pass USE_CHIME_SDK_MEETINGS otherwise it should be false. SO instead of passing -u flag to determine whar chime client to use we should use USE_CHIME_SDK_MEETINGS parameter

Reply:
I think you meant set value of constant useChimeSDKMeetings to false, but when CHIME_SDK_MEETINGS_ENDPOINT is passed, set the value to true